doc/jsk_perception/nodes/label_image_classifier.rst

```diff
@@ -0,0 +1,42 @@
+label_image_classifier.py
+=========================
+
+
+What is this?
+-------------
+
+.. image:: ./images/label_image_classifier.png
+
+Classify from label image.
+
+Return largest segmented label.
```

Comment:
what is the definition of 'largest' ? number of pixels? number of pixels times probability?